services/MerchStoreService.ts

```diff
@@ -457,4 +472,12 @@ export default class MerchStoreService {
     }
     return requestedQuantitiesByMerchItem;
   }
+
+  private getTotalCost(order:MerchItemOptionAndQuantity[], itemOptions:Map<string, MerchandiseItemOptionModel>):number {
```

Comment:
Nit: this can be `private static totalCost(...)` since it only uses its parameters and no class variables.